worker/ffmpeg_worker.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A server that runs FFmpeg on files in Google Cloud Storage.
+"""
+
+import os
+import subprocess
+from typing import Tuple
+
+from flask import Flask
+from flask import request
+from flask import safe_join
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+app = Flask(__name__)
+
+
+@app.errorhandler(NotFound)
+def _handle_gcloud_not_found(_):
+    return 'Google Cloud storage object not found', 404
+
+
+@app.route('/ffmpeg', methods=['POST'])
+def process_ffmpeg_command():
+    """Runs ffmpeg according to the request's specification.
+
+    The POST request supports the following arguments:
+        input_file: the input file in the Google Cloud Bucket specified
+                    Example: bucket_name/path/to/file
+        output_file: the output file that should be stored in the
+                     Google Cloud Bucket specified
+                     Example: bucket_name/path/to/file
+    This route changes the video container format by running the following:
+        ffmpeg -i {input_file} {output_file}
+
+    Args:
+
+    Returns:
+        A string with the ffmpeg logs.
+    """
+    remote_input_file = request.form['input_file']
+    local_input_file = _localize_filename(remote_input_file)
```

Comment:
Use the functions in python tempfile library https://docs.python.org/3.5/library/tempfile.html to create tempfiles for input and output  instead of using  /tmp/*  .